src/main/java/com/booksaw/betterTeams/team/meta/TeamMeta.java

```diff
@@ -0,0 +1,84 @@
+package com.booksaw.betterTeams.team.meta;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+
+/**
+ * This class manages custom metaData for a team.
+ * It stores various properties as key-value pairs.
+ */
+public class TeamMeta {
+
+	private final Map<String, String> metaData;
+
+	public TeamMeta() {
+		this.metaData = new HashMap<>();
+	}
+
+	/**
+	 * Adds or updates a metaData entry.
+	 * @param key   The identifier for the metaData.
+	 * @param value The string value of the metaData.
+	 */
+	public boolean set(String key, String value) {
+		this.metaData.put(key, value);
+		return true; // for now
```

Comment:
in what case would this fail in the future? If we don't expect this to ever fail does it really need to return anything?